index.html

```diff
@@ -3,19 +3,26 @@
   <head>
     <meta charset="UTF-8" />
     <title>Test</title>
-    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"
+    />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
```

Comment:
Do you really need `materialize.min.js` here?

Reply:
I dont kow, i wont to do fast